homeassistant/components/saj/sensor.py

```diff
@@ -230,4 +235,7 @@ def async_update_values(self, unknown_state=False):
     @property
     def unique_id(self):
         """Return a unique identifier for this sensor."""
+        if self._inverter_name:
+            return f"{self._inverter_name}_{self._sensor.name}"
```

Comment:
We don't want the unique_id to be configurable since the unique_id should be stable.

Reply:
I understand your point of view but how can we achieve this when users want to add multiple "instances" of this platform?
Or do you mean it should contain the host/ip instead of a custom name so it's always the same unique id in combination with the same device?

Reply:
It should be a serial of the device or mac address eg. IP addresses may not be stable either.

Reply:
Agh, sorry @MartinHjelmare.

Reply:
Ok guys, I've managed to get the serial number of the device so I'm able to link it to the unique id.
Since this PR is closed, I suppose I need to create a new one and refer to tis one?

Reply:
@MartinHjelmare @bachya 
I've created a new PR (#28663) with the changes necessary for a unique id based on device serial number. Thanks for reviewing.